config/base/kustomization.yaml

```diff
@@ -1,5 +1,29 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../default
   - ../prometheus
+  - ../overlays/odh
+
+namespace: opendatahub
+configMapGenerator:
+  - envs:
+      - params.env
+    name: odh-model-controller-parameters
+generatorOptions:
+  disableNameSuffixHash: true
+
+vars:
+  - fieldref:
+      fieldPath: metadata.namespace
+    name: mesh-namespace
+    objref:
+      apiVersion: v1
+      kind: ConfigMap
+      name: odh-model-controller-parameters
```

Comment:
Where is `mesh-namespace` being used?

Reply:
leftover `fieldref` from the common kustomization file for modelmesh and odh model controller from `odh-manifests`. Removed :)